packages/mux-uploader/test/uploader.test.js

```diff
@@ -1,5 +1,69 @@
-import { fixture, assert, aTimeout } from '@open-wc/testing';
-/** @todo Need to identify and resolve why upchunk causes issues in our testing environment. (TD). */
-// import '../src/index.ts';
+import { fixture, assert, oneEvent } from '@open-wc/testing';
+import '../src/index.ts';
 
-describe('<mux-uploader>', () => {});
+describe('<mux-uploader>', () => {
+  let file = new File(['foo'], 'foo.mp4', {
+    type: 'video/mp4',
+  });
+
+  it('initiates as expected', async function () {
+    const uploader = await fixture(`<mux-uploader
+      endpoint="https://my-authenticated-url/storage?your-url-params"
+      status
+    ></mux-uploader>`);
+
+    assert.equal(uploader.getAttribute('type'), 'bar', 'default progress type is reflected');
+    assert.equal(
+      uploader.getAttribute('endpoint'),
+      'https://my-authenticated-url/storage?your-url-params',
+      'endpoint matches'
+    );
+    assert.equal(uploader.getAttribute('status'), '', 'status matches');
+  });
+
+  it('does not init without endpoint', async function () {
+    const uploader = await fixture(`<mux-uploader
+      status
+    ></mux-uploader>`);
+
+    setTimeout(() => {
+      uploader.dispatchEvent(
+        new CustomEvent('file-ready', {
+          composed: true,
+          bubbles: true,
+          detail: file,
+        })
+      );
+    });
+
+    const { detail } = await oneEvent(uploader, 'uploaderror');
+
+    assert.equal(detail.message, 'No url or endpoint specified -- cannot handleUpload', 'error message matches');
+    assert.exists(uploader.getAttribute('upload-error'), 'upload error is true');
+  });
+
+  it('completes a mock upload', async function () {
+    const uploader = await fixture(`<mux-uploader
+      endpoint="/mock-upload-endpoint"
```

Comment:
suggestion(non-blocking): Add a code comment here referencing the dev server middleware config dependency.